ee/clickhouse/models/team.py

```diff
@@ -0,0 +1,31 @@
+import logging
+from typing import List
+
+from ee.clickhouse.client import sync_execute
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+logger = logging.getLogger(__name__)
+
+# Note: Session recording, dead letter queue, logs deletion will be handled by TTL
+TABLES_TO_DELETE_FROM = [
+    "events",
+    "person",
+    "person_distinct_id",
+    "person_distinct_id2",
+    "groups",
+    "cohortpeople",
+    "person_static_cohort",
+]
+
+
+def delete_teams_data(team_ids: List[int]):
+    logger.info(
+        f"Deleting teams data from clickhouse using background mutations.",
```

Comment:
Worth adding in a log for each table? We can then also get an understanding of how long each takes

Reply:
Deletion is async so this will return immediately.

Reply:
Ah ok, makes sense